runtime/autoload/health/provider.vim

```diff
@@ -240,6 +240,11 @@ function! s:check_python(version) abort
   let pyname = 'python'.(a:version == 2 ? '' : '3')
   let pyenv = resolve(exepath('pyenv'))
   let pyenv_root = exists('$PYENV_ROOT') ? resolve($PYENV_ROOT) : ''
+  let pyenv_flag = 0
+  if empty(pyenv_root)
+    let pyenv_root = !empty(pyenv) ? s:trim(s:system([pyenv, 'root'])) : ''
+    let pyenv_flag = 1
```

Comment:
instead of an extra flag, can this be done in the `if empty(pyenv_root)` block on line 260?

Reply:
You mean at at line 260 check `if empty(pyenv_root)` first, if it's empty then try to set it as the output of `pyenv root`? I think that works! Thanks:)